csharp/src/Ice/Endpoint.cs

```diff
@@ -52,15 +34,18 @@ public abstract class Endpoint : IEquatable<Endpoint>
         /// <value>True when this endpoint's transport is secure; otherwise, false.</value>
         public abstract bool IsSecure { get; }
 
-        /// <summary>The name of the endpoint's transport in lowercase, or "opaque" when the transport's name is
-        /// unknown.</summary>
-        public virtual string Name => Type.ToString().ToLowerInvariant();
+        /// <summary>The Ice protocol of this endpoint.</summary>
+        public Protocol Protocol { get; }
 
         /// <summary>The timeout for the endpoint in milliseconds. 0 means non-blocking, -1 means no timeout.</summary>
         public abstract int Timeout { get; }
 
-        /// <summary>The <see cref="EndpointType">type</see> of this endpoint.</summary>
-        public abstract EndpointType Type { get; }
+        /// <summary>The <see cref="Transport"></see> of this endpoint.</summary>
```

Comment:
```suggestion
        /// <summary>The <see cref="ZeroC.Ice.Transport"/> of this endpoint.</summary>
```
Seems that otherwise the link points to this property

Reply:
Fixed.